INSTALL.rst

```diff
@@ -56,6 +56,15 @@ You need to add ``~/.local/bin`` to the front of your ``$PATH``:
   echo 'export PATH="$HOME/.local/bin:$PATH' >> ~/.bashrc
   source ~/.bashrc
 
+On MacOSX you also need to add ~/.local/lib/pythonX.Y/site-packages to your
```

Comment:
I tested it on ubuntu 16.04 +python3 don't need to define any additional environment variable